tap_hubspot/__init__.py

```diff
@@ -522,9 +609,14 @@ def sync_deals(STATE, ctx):
         params['includeAllProperties'] = True
         params['allPropertiesFetchMode'] = 'latest_version'
 
+        # Grab selected `hs_date_entered/exited` fields to call the v3 endpoint with
+        v3_fields = [x[1].replace('property_', '')
+                     for x,y in mdata.items() if x and (y.get('selected') == True or has_selected_properties)
```

Comment:
Niptick, but please do not name these x,y. Name them something longer so I understand better what information they represent.

Reply:
Done